packages/sitecore-jss/src/graphql/graphql-edge-proxy.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Generates a URL for accessing Sitecore Edge Platform Content using the provided endpoint and context ID.
+ * @param {string} endpoint - The base endpoint URL for the Edge Platform.
+ * @param {string} contextId - The unique context ID.
+ * @returns {string} The complete URL for accessing content through the Edge Platform.
+ */
+export const getEdgeProxyContentUrl = (endpoint: string, contextId: string) =>
```

Comment:
As discussed, we should swap these params and make endpoint optional, falling back to https://edge-platform.sitecorecloud.io. 

Also, we should name these parameters to follow our standard naming (`sitecoreEdgeUrl`, `sitecoreEdgeContextId`)